src/qml/pages/onboarding/Onboarding00Welcome.qml

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The Onboarding00Welcome page.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+Pane {
+    background: Rectangle {
```

Comment:
Not needed? The window background is already black. To improve rendering time make `background: null`

Reply:
Documentation for null or for rendering improvement?